src/main/java/com/milaboratory/mixcr/cli/ActionAlign.java

```diff
@@ -121,6 +117,20 @@ public void go(ActionHelper helper) throws Exception {
 
         System.out.println("Reference library: " + library.getLibraryId());
 
+        // Printing library level warnings, if specified for the library
+        if (!library.getWarnings().isEmpty()) {
+            System.out.println("Library warnings:");
+            for (String l : library.getWarnings())
+                System.out.println(l);
+        }
+
+        // Printing citation notice, if specified for the library
+        if (!library.getCitations().isEmpty()) {
+            System.out.println("Please cite:");
```

Comment:
"Please cite:" or some other text? add option ?

Reply:
Like specify exact message in library meta field? I think fixed message is ok.

Reply:
This implies that "citation" in library is given in exactly in _citation format_ (should be documented), since someone may add "Please cite PaperXXX" text already in the library citation message.

Reply:
👍

Reply:
Added https://github.com/repseqio/repseqio/issues/38